src/main/xsd/state/1.0.0/types.xsd

```diff
@@ -138,6 +138,26 @@
     </xs:restriction>
   </xs:simpleType>
 
+	<xs:complexType name="TimeIntervalType">
```

Comment:
I don't wish to elicit a holy war here, but... replace tabs with spaces please.

Reply:
Urgh. Sorry, copied it in from xmlspy and didn't notice the difference in vscode...